homeassistant/components/dynalite/config_flow.py

```diff
@@ -32,27 +30,12 @@ def __init__(self):
         """Import a new bridge as a config entry."""
         LOGGER.debug("async_step_import - %s", import_info)
         host = self.context[CONF_HOST] = import_info[CONF_HOST]
```

Comment:
Where do we use the context?

Reply:
I am quite a newbie in these config flows. I just copied it from an example. I don't use it explicitly. Can I just remove this part?

Reply:
Yes. It looks like we can remove it.

Reply:
removed